reference/orders.v2.oas2.yml

```diff
@@ -2974,6 +3010,9 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/orderProductOptions'
+        configurable_fields: 
```

Comment:
If someone could provide the description/schema for this field that would be great

Reply:
it can be:
```
configurable_fields:
          type: array
          items:
            type: object
            properties:
              name:
                type: string
                example: Engraving
              value:
                type: string
                example: Rabin
```

Reply:
Thanks, I just added the schema. How can people configure these fields? I didn't see anything in the control panel

Reply:
I believe it's the catalog v2 fields, you would need to enable catalog v2 in your store to see it by disabling `Feature_NewAddEditProduct` and `NewAddEditForNewTrials`

Reply:
Thanks, I've edited the description